source/support/check-organisation-email-address.html.erb

```diff
@@ -0,0 +1,47 @@
+---
+title: Check your organisation email address - GovWifi
+description: Check your government or public sector email address to sign up to GovWifi.
+---
+<div class="container">
+  <div class="grid-row">
+    <div class="column-one-third">
+      <%= partial 'shared/support/sidebar' %>
+    </div>
+    <div class="column-two-thirds">
+      <h1>Check your organisation email address</h1>
+      <p>You can <%= link_to "signup to GovWifi", "/support/connect-to-govwifi", class: "govuk-link" %>
+        using your government or public sector email address
+      </p>
+      <p>If you don't have a government or public sector email address, you can
+        <%= link_to "connect to GovWifi as a visitor", "support/connect-visitors-to-govwifi", class: "govuk-link" %>
+      </p>
+      <p>You can check to see if your organisation email is eligible below:</p>
+      <div class="govuk-form-group">
+        <input class="govuk-input input-field-margin-bottom" id="search-table" type="text" placeholder="Search email domains">
+      </div>
+
+      <% domains = data.domains.sort %>
+
+      <table class="govuk-table">
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th class="govuk-table__header govuk-!-width-one-half">Organisation email domain name</th>
+          </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+          <% domains.each do |email| %>
+            <tr class="govuk-table__row result-row results">
+              <td class="govuk-table__cell govuk-!-width-one-half" scope="row">
```

Comment:
What I was saying earlier was that we should add a class here so we can target it.
              <td class="govuk-table__cell govuk-!-width-one-half **search-filter-by**" scope="row">

This reduces the risk of someone trying to change styles and breaking this selector for filtering.

Reply:
It does have a class called results?